chapters/dashboard/self-host.md

```diff
@@ -0,0 +1,21 @@
+## Self-host a Dashboard
```

Comment:
For this page, can you try using this template? https://github.com/jina-ai/docs/pull/21 

I know your approach is simpler, but it's an attempt to have a standard look throughout the docs.

Reply:
```suggestion
## Self-host Dashboard
```